### PR TITLE
Add centrally deployed packages.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: sysconfig/bb5/users/packages.yaml [hunk 1]
```diff
@@ -1,188 +1,287 @@
...
+    paths:
+      binutils@2.29.1: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/2018-09-28/install/linux-rhel7-x86_64/gcc-6.4.0/binutils-2.29.1-quhkkp
+  bison:
+    paths:
+      bison@3.0.4: /usr
+    version: [3.0.4]
+  boost:
+    paths:
+      boost@1.67.0: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/boost-1.67.0-6skhmz
+      boost@1.67.0 ~graph: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/boost-1.67.0-6skhmz
```
Comment (pramodk) [inline sysconfig/bb5/users/packages.yaml:20]: do you remember why +graph was a problem?

File: sysconfig/bb5/users/packages.yaml [hunk 2]
```diff
@@ -1,188 +1,287 @@
...
+      py-arrow@0.10.0: /activated
+  py-cycler:
+    paths:
+      py-cycler@0.10.0: /activated
+  py-dateutil:
+    paths:
+      py-dateutil@2.5.2: /activated
+  py-h5py:
+    paths:
+      py-h5py@2.7.1+mpi: /activated
```
Comment (pramodk) [inline sysconfig/bb5/users/packages.yaml:182]: should I change this ~mpi?

File: sysconfig/bb5/users/packages.yaml [hunk 3]
```diff
@@ -1,188 +1,287 @@
...
+  gtkplus:
+    paths:
+      gtkplus@2.24.31: /usr
+    version: [2.24.31]
+  # hdf5:
+  #   paths:
+  #     hdf5@1.8.12~mpi~hl: /usr
+  #     hdf5@1.10.2+hl+mpi+shared+szip: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.2-dueuws
+  #     hdf5@1.10.2 +hl+szip: /gpfs/bbp.cscs.ch/apps/hpc/spack-deployments/18-06-2018/install/linux-rhel7-x86_64/gcc-6.4.0/hdf5-1.10.2-dueuws
+  #   version: [1.8.12, 1.10.2]
```
Comment (pramodk) [inline sysconfig/bb5/users/packages.yaml:75]: having hdf5 is necessary....what you suggest here? I can think of providing +mpi and ~mpi installations.

File: sysconfig/bb5/users/packages.yaml (hunk 2 above)
Comment (matz-e) [reply sysconfig/bb5/users/packages.yaml:182]: I think that this h5py actually is compiled with MPI support, I'd leave it in.

File: sysconfig/bb5/users/packages.yaml (hunk 3 above)
Comment (matz-e) [reply sysconfig/bb5/users/packages.yaml:75]: Yes, having both by default would be good.

File: sysconfig/bb5/users/packages.yaml (hunk 1 above)
Comment (matz-e) [reply sysconfig/bb5/users/packages.yaml:20]: I don't, not sure if I was involved with that. This may have been picked up automatically…

**Comment by ferdonline** [comment]
Just for curiosity, this is a new full stack we are deploying? So even the tools (cmake...) are not the system ones?

File: sysconfig/bb5/users/packages.yaml (hunk 3 above)
Comment (matz-e) [reply sysconfig/bb5/users/packages.yaml:75]: I think this failed for me for some weird reason, and I decided that a manual rebuild would be faster. Maybe swapping the version preferences should be enough?

File: sysconfig/bb5/users/packages.yaml (hunk 2 above)
Comment (ferdonline) [reply sysconfig/bb5/users/packages.yaml:182]: Related to `variant: ~mpi` to pick the non mpi version by default?

Comment (pramodk) [reply sysconfig/bb5/users/packages.yaml:182]: yeah...I also think better to not have mpi involved

**Comment by pramodk** [comment]
@ferdonline:  this is intermediate one for deploying spykfunc and parquet-converter modules.

**Comment by matz-e** [comment]
I think we can close this now.

**Pull request closed without merging**